### DISCUSSION TITLE
How to test the "login page expired" template?

### DISCUSSION THREAD

**Discussion by sgrimm** [keycloak/keycloak#15988] [Q&A]
I have a custom theme and want to make sure it is tested before it gets deployed to end users.

For most of the templates, this is straightforward, but I can't figure out how to get Keycloak to use the `login/login-page-expired.ftl` template. If I sit on the login page for longer than the realm's login timeout and then submit the login form, I get `login/login.ftl` again with an error message about the login having expired.

What do I need to do to force Keycloak to show me a page using that template?

**Comment by Xavier-Wattermann** [comment]
Couple things to try:
 - You can modify the timeout settings in the realm settings -> tokens (which makes it easier to test)
 - If you send a user a few credential reset actions (CONFIGURE TOTP, UPDATE PASSWORD, as an example), go through the configure totp page, and **then hit the back button in the browser** that can reliably trigger the `login-page-expired` page for me. 

Credential Reset is found in Users -> Individual User -> Credentials -> Credential Reset
